src/app/components/TradingChart/helpers.ts

```diff
@@ -0,0 +1,284 @@
+import { ApolloClient, gql } from '@apollo/client';
+import {
+  CandleDetails,
+  CandleDuration,
+} from 'app/pages/PerpetualPage/hooks/graphql/useGetCandles';
+import { Asset } from 'types';
+import { getTokenContract } from 'utils/blockchain/contract-helpers';
+import { TradingCandleDictionary } from './dictionary';
+
+export const MAX_CANDLE_COUNT = 500;
+export const graph_endpoint: string = process.env.REACT_APP_GRAPH_RSK!;
```

Comment:
we have this variable in classifiers (graphRskUrl) and by the way where you use it?

Reply:
you are right, it's not used anymore. Removing it